plugins/api/src/atomic_apply_acl_operations.cpp

```diff
@@ -496,8 +496,15 @@ namespace
 
         rodsLog(LOG_DEBUG, "Checking if user has permission to modify permissions ...");
 
-        if (!ic::user_has_permission_to_modify_acls(*_comm, db_conn, db_instance_name, object_id)) {
-            rodsLog(LOG_ERROR, "User not allowed to modify ACLs [logical_path=%s, object_id=%d]", logical_path.data(), object_id);
+        if (const auto iter = input.find("admin_mode"); iter != std::end(input) && iter->get<bool>()) {
+            if (!irods::is_privileged_client(*_comm)) {
+                rodsLog(LOG_ERROR, "User is not an administrator.");
```

Comment:
I see that this is the same message as atomic_apply_metadata, but I figured I would just throw this out there :p

Would this benefit from a little more context in the log message? Just thinking about seeing this message appear in the logs and wondering what it could mean.

Maybe something like...

admin_mode cannot be enabled in rs_atomic_apply_acl_operations because the user is not an administrator.

Reply:
Yes. Will add more words.

Reply:
Done.